tsconfig.eslint.json

```
@@ -0,0 +1,8 @@
+{
+	"extends": "./tsconfig.base.json",
+	"compilerOptions": {
+		"allowJs": true,
+		"checkJs": true
+	},
+	"include": ["src", "test", "examples", "jest.config.js", "rollup.config.ts"]
```

```suggestion
	"include": ["src", "test", "examples", "jest.config.js"]
```

There is no rollup config